scrapers/KristenBjorn.yml

```diff
@@ -0,0 +1,114 @@
+name: "KristenBjorn"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/video
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/web/model/store/
+    scraper: movieScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/gay-porn-star/
+      - kristenbjorn.com/web/model/gay-porn-star/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="scene-item"]/a[@class="scene-box"]
+    scene:
+      Title: //article/div[1]//h2/text()
+      Details: //div[@class="med-text"]/p/text()
+      Tags:
+        Name: //a[@role="button"]/text()
+      Performers:
+        Name: $performer//h5/text()
+        URL:
+          selector: $performer/@href
+          postProcess: &appendDomain
+            - replace:
+                - regex: ^(.*)$
+                  with: http://kristenbjorn.com$1
+      Image: &imageAttr //meta[@property="og:image"]/@content
+      Studio:
+        Name:
+          fixed: Kristen Bjorn
+      Movies:
+        Name: //article/div[5]//h2/text()
+        URL:
+          selector: '//table[@style="width: 90%;margin-top: 20px;"]//img/parent::a/@href'
+          postProcess: *appendDomain
+  movieScraper:
+    movie:
+      Name: //article/div[1]/span/text()
+      Date:
+        selector: //span[text()="Release Date:"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: /19([0-9])
+                with: /$1
+              - regex: /20([0-9])
+                with: /$1
+          - parseDate: 01/02/06
+      Duration:
+        selector: //span[text() = "Running Time:"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: \sh(ou)?rs?\s?
+                with: ':'
+              - regex: \smin(ute)?s?
+                with: ':00'
+              - regex: :$
+                with: :00:00
```

Comment:
this doesnt relay work,  set log to debug and check when you do some scrapes
i tried the below
```yaml
          - replace:
              - regex: ^(\d+)\s*(?i:h)\D+$ # movies with only hours
                with: $1:00:00
              - regex: ^(\d+)\s*(?i:m)\D+$ # movies with only minutes
                with: '$1:00'
              - regex: ^(\d+)\s*(?i:h)\D+(\d+)\s*(?i:m)\D+$ # movies with hours and minutes
                with: '$1:$2:00'
```
and it seems to work for the 3 different cases listed, if  there are more cases we need to check that again

Reply:
Thanks! This seems to work too with the movies I was trying.